src/types/models/buyerFeedbackBatch.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiSearchListing } from "../../types/models/listing"
+
+export interface BuyerFeedbackDealer {
+  id: number
+  name: string
+  location: {
+    city: string
+    zipCode: string
+  }
+}
+
+export interface BuyerFeedbackListing
+  extends Pick<ApiSearchListing, "id" | "make" | "model" | "price" | "type"> {
```

Comment:
can you quickly explains what this TS magic does?

Reply:
With this utility `Pick<Type, Keys>` we pass an existing Type and set the keys that we need from the Type. Therefore, there's no need to repeat again the properties that we have already declared in another Type.

Reply:
got it. so we end up exposing a Listing-ish but with a limited amount of fields